change-beta/@azure-communication-react-b4bd7243-4950-4fee-b9c5-6c42c93b84a4.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "prerelease",
+  "area": "feature",
+  "workstream": "PowerPoint Live - Teams Interop",
+  "comment": "We are excited to announce that the Azure Communication Services Web UI Library now can view PowerPoint Live sessions initiated by a Teams client. Users can follow along with the current slide the presenter is sharing and view presenter annotations. Developers can use this functionality today through our composites (e.g CallComposite, CallWithChatComposite) as well as through components (e.g VideoGallery).",
+  "packageName": "@azure/communication-react",
+  "email": "93549644+ShaunaSong@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
you need to change this to pre-release

Reply:
is this going to GA? I would argue its a `minor` change if so.

Reply:
this is what we have in our documentation

![image](https://github.com/Azure/communication-ui-library/assets/79329532/37af8c84-b3a0-4eab-81d4-58c75bfa168c)

Reply:
This looks like its just for inProgress->beta, we should have different instructions for beta->stable